tests/integration/src/rust_masm_tests/rust_sdk.rs

```diff
@@ -215,12 +188,16 @@ fn rust_sdk_cross_ctx_note() {
     test.expect_masm(expect_file![format!("../../expected/rust_sdk/{artifact_name}.masm")]);
     let package = test.compiled_package();
     let mut exec = Executor::new(vec![]);
-    let account_package =
-        Arc::new(Package::read_from_bytes(&std::fs::read(masp_path).unwrap()).unwrap());
-    exec.dependency_resolver_mut()
-        .add(account_package.digest(), account_package.into());
+    for dep_path in test.dependencies {
+        let account_package =
+            Arc::new(Package::read_from_bytes(&std::fs::read(dep_path).unwrap()).unwrap());
+        exec.dependency_resolver_mut()
+            .add(account_package.digest(), account_package.into());
+    }
     exec.with_dependencies(&package.manifest.dependencies).unwrap();
-    let trace = exec.execute(&package.unwrap_program(), &test.session);
+    // TODO: uncomment after Paul's fix for mem intrinsics (failing u32 assert) is merged
```

Comment:
The fix for this was merged, so this should be able to be uncommented now

Reply:
Thanks! Done.